dashboard/app/models/levels/blockly.rb

```diff
@@ -457,11 +457,18 @@ def localized_failure_message_override
     get_localized_property('failure_message_override')
   end
 
+  # Retrieve the localized property for "long_instructions",
+  # @return [String] the localized long_instructions property
   def localized_long_instructions
     localized_long_instructions = get_localized_property("long_instructions")
-    localized_blockly_in_text(localized_long_instructions)
+    unescaped_codeblocks = unescape_codeblocks(localized_long_instructions)
+    localized_blockly_in_text(unescaped_codeblocks)
   end
 
+  # Processes and localizes the TRANSALTIONTEXT from i18n start libraries content.
```

Comment:
typo TRANSLATIONTEXT
```suggestion
  # Processes and localizes the TRANSLATIONTEXT from i18n start libraries content.
```